src/vmstats_server.erl

```diff
@@ -19,43 +19,31 @@
                 prev_io :: {In::integer(), Out::integer()},
                 prev_gc :: {GCs::integer(), Words::integer(), 0}}).
 %%% INTERFACE
-start_link() ->
-    start_link(base_key()).
-
-%% the base key is passed from the supervisor. This function
-%% should not be called manually.
-start_link(BaseKey) ->
-    gen_server:start_link({local, ?MODULE}, ?MODULE, BaseKey, []).
+start_link(Sink, BaseKey) ->
+    gen_server:start_link({local, ?MODULE}, ?MODULE, {Sink, BaseKey}, []).
 
 %%% INTERNAL EXPORTS
-init(BaseKey) ->
+init({Sink, BaseKey}) ->
     {ok, Delay} = application:get_env(vmstats, delay),
     Ref = erlang:start_timer(Delay, self(), ?TIMER_MSG),
-    {{input,In},{output,Out}} = erlang:statistics(io),
-    PrevGC = erlang:statistics(garbage_collection),
+    {{input, In}, {output, Out}} = erlang:statistics(io),
+    {GCs, Words, _} = erlang:statistics(garbage_collection),
```

Comment:
uh, not sure why I ever carried the 0 there; it was documented as 0 as far as R15B.